libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -379,6 +382,14 @@ export class MultiInputComponent implements
             this.popoverRef.refreshPosition();
         }
 
+        this._resetSearchTerm();
+
+        if (this.popoverRef) {
+            this.popoverRef.close();
+        }
+
+        this.searchInputElement.nativeElement.focus();
```

Comment:
Why do you close the popover once an option gets selected?
Shouldn't we allow multi-selection?

Reply:
I was going off of sapui5's behavior, but looking at it again today, it appears as though they keep the popover open if you click one of the checkboxes but close it if you click any other part of the list item.  I doubt that is actually the intended behavior so I will remove this line